docs/opcodes/FD.mdx

```diff
@@ -5,18 +5,30 @@ group: System operations
 
 ## Notes
 
-Equivalent to [CREATE](/#F0), except the salt allows to create at a deterministic address.
+Abort execution, undo all state changes, return a value and refund the remaining gas to the caller.
+The error message returned is available to the caller in the returndata buffer.
 
 ## Stack input
 
-0. `Integer`: value in [wei](https://www.investopedia.com/terms/w/wei.asp) to send to the new account.
-1. `Integer`: offset in the [memory](/about) in bytes, the instructions of the new account.
-2. `Integer`: how many bytes to copy (size of the instructions).
-3. `Integer`: salt, 32-byte value used to create the new account at a deterministic address.
+0. `Integer`: offset in the [memory](/about) in bytes. The starting position of the value to return.
+1. `Integer`: length in the [memory](/about) in bytes to return.
+
+## Returndata output
+
+memory[offset:offset+length]
+
+## Examples
+
+| Memory |
```

Comment:
Here you can add another array (at the bottom) to show the actual bytes of the return data according to the given example memory and parameters. You can call it "Calling context return data"